region/maridia/inner-green/East Sand Hall.json

```
@@ -1757,7 +1882,72 @@
           ]}
         ]}
       ],
+      "flashSuitChecked": true,
       "note": "To save Energy, spark diagonally to get onto the pillar, then jump to the door."
+    },
+    {
+      "link": [5, 1],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 53, "excessFrames": 8}}
+      ],
+      "flashSuitChecked": true,
+      "devNote": [
+        "With a controlled shinespark end, starting with 74 energy, this can collect an Evir drop."
+      ]
+    },
+    {
+      "link": [5, 2],
+      "name": "Base",
+      "requires": [],
```

Doesn't really matter, since you shouldnt be able to get here without one of these. But it looks like a nontrivial jump
```suggestion
      "requires": [
        "h_navigateUnderwater"
      ],
```